examples/xa-datasource/README_pt.adoc

```diff
@@ -0,0 +1,334 @@
+Title: Injeção do EntityManager
```

Comment:
add the following header:

:index-group: DataSources
:jbake-type: page
:jbake-status: status=published
= Injection Of Entitymanager

Reply:
Is corrected

Reply:
please remove this -> Title: Injeção do EntityManager

Reply:
and fix   = Injection Of Entitymanager to =  Injeção do EntityManager